drivers/sensor/st/iis328dq/iis328dq_trigger.c

```diff
@@ -175,8 +175,10 @@ static void iis328dq_handle_interrupt(const struct device *dev)
 	struct iis328dq_data *data = dev->data;
 	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
 	iis328dq_status_reg_t status;
+#ifdef CONFIG_IIS328DQ_THRESHOLD
```

Comment:
The variables should probably be marked as `__maybe_unused`. See https://www.kernel.org/doc/html/latest/process/coding-style.html section 21.

Reply:
Thank you for your review.
I applied your suggestion.

Reply:
Zephyr convention is to use the ifdef